cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -49,12 +51,17 @@ if(ANDROID)
   else()
     hunter_user_error("Could not find android architecture. Please set ANDROID_ARCH_NAME in your toolchain or use CMake 3.7+")
   endif()
-  set(configure_command
-      # Ignored. Prevents script from checking host uname
-      RELEASE=2.6.37
-      SYSTEM=android
-      ARCH=${ANDROID_SSL_ARCH}
-      ${configure_command})
+
+  # Latest OpenSSL (1.1.1) still does not have correct android targets, so set generic with our CFLAGS
+  # See: https://github.com/openssl/openssl/blob/master/Configure
```

Comment:
I don't see any notes that this file is for Android. Also please use stable links.